infra/feast-operator/internal/controller/services/services_types.go

```diff
@@ -67,15 +66,15 @@ var (
 
 	FeastServiceConstants = map[FeastServiceType]deploymentSettings{
 		OfflineFeastType: {
-			Command:    []string{"feast", "serve_offline", "-h", "0.0.0.0"},
+			Command:    []string{"feast", "--log-level", "DEBUG", "serve_offline", "-h", "0.0.0.0"},
```

Comment:
though we probably need to make this configurable in the CR

Reply:
Agree! I already suggested you to add a new story/task under the epic for troubleshooting support, this would definitely help as logging is really poor at the default level.